### ISSUE TITLE
Rearrange does nothing in Sketch 46.2

### ISSUE THREAD

**Issue by kyleighsmith** [romannurik/Sketch-ArtboardTricks#20]
Not working with Sketch 46.2. Try selecting re-arrange and nothing happens.

**Comment by UIGirl** [comment]
Hi Roman, I love this plugin and I miss using it! I can confirm that it's not working with Sketch 46.2
When I select re-arrange nothing happens.
I want to thank you for this plug in, it makes my life easier. I hope that you have time to update it soon. :)

**Comment by romannurik** [comment]
Is this the same as #18, where on High Sierra it crashes?

**Comment by romannurik** [comment]
FWIW, I can't reproduce this in either Sketch 46.2 nor Sketch 47 on macOS 10.12 Sierra.

**Comment by kieferpartridge** [comment]
@romannurik Running Sketch 47.1 on El Capitan and can't get it to work either.
But it's working just fine on another machine with Sierra/Sketch 46.

**Comment by romannurik** [comment]
Could you open the Console Mac app, do a search for "sketch", run the
rearrange command in Sketch, and attach the output you see in Console to
this bug? Basically want to see if there are any errors when running the
command.

**Comment by kieferpartridge** [comment]
@romannurik Here you go — looks like it might be pretty simple, actually.
```
10/19/17 2:02:21.031 PM Rearrange Artboards Into Grid (Sketch)[38630]: SyntaxError: Unexpected token '>'.
Plugin “rearrange_artboards_into_grid”, line 12:
»   layers.forEach(l => l.select_byExpandingSelection_(true, true));

10/19/17 2:02:21.037 PM Sketch[38630]: Exception: {
    line = 12;
    sourceURL = "/Users/a541836/Library/Application Support/com.bohemiancoding.sketch3/Plugins/ArtboardTricks.sketchplugin/Contents/Sketch/rearrange_artboards_into_grid.js";
    stack = undefined;
}
```

**Comment by romannurik** [comment]
Got it. This looks related to ES6 not being supported on El Capitan. I've got a fix planned.

**Comment by romannurik** [comment]
Can y'all try this build to see if it works?

[plugin.zip](https://github.com/romannurik/Sketch-ArtboardTricks/files/1399905/plugin.zip)

**Comment by kieferpartridge** [comment]
Works like a charm in Sketch 47/El Capitan. Brilliant.

**Comment by romannurik** [comment]
Sweet. Will be fixed properly in next update

**Comment by UIGirl** [comment]
That build worked for me! Yay! I have nicely organized artboards again. Satisfying. 

Thanks Roman!

**Comment by romannurik** [comment]
Fixed in 1.0.7
